src/domains/idpay-common/03_database.tf

```diff
@@ -541,10 +541,6 @@ locals {
         keys   = ["_id"]
         unique = true
         },
-        {
```

Comment:
@Benedetta-fabbri @dariopelliccioli unique index can be modified only when collection is empty. I'm getting the following error

```
│ Error: waiting on create/update future for Cosmos Mongo Collection "merchant" (Account: "cstar-u-idpay-mongodb-account", Database: "idpay"): Code="Forbidden" Message="Error=13, Details='Response status code does not indicate success: Forbidden (403); Substatus: 0; ActivityId: 988adc02-db55-4c08-b88c-78a9e2dcd980; Reason: (Message: {\"Errors\":[\"Unique Index can be modified only if collection is empty\"]}
```

Reply:
Sorry, it is now empty.